### PR TITLE
Make settings names translatable

### PR DESCRIPTION
**Pull request by urbalazs**
There are much more items (such as <title> or <name>) which may be translatable, but I'm not sure about that. Please review it!

### CODE REVIEW

File: Kernel/Config/Files/FAQ.xml [hunk 1]
```diff
@@ -6,9 +6,9 @@
         <SubGroup>Interface</SubGroup>
         <Setting>
             <Array>
-                <Item>internal</Item>
-                <Item>external</Item>
-                <Item>public</Item>
+                <Item Translatable="1">internal</Item>
+                <Item Translatable="1">external</Item>
+                <Item Translatable="1">public</Item>
             </Array>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: Strings here should not be translatable as they are the DB names of the state types.

File: Kernel/Config/Files/FAQ.xml [hunk 2]
```diff
@@ -18,8 +18,8 @@
         <SubGroup>Interface</SubGroup>
         <Setting>
             <Array>
-                <Item>external</Item>
-                <Item>public</Item>
+                <Item Translatable="1">external</Item>
+                <Item Translatable="1">public</Item>
             </Array>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: Strings here should not be translatable as they are the DB names of the state types.

File: Kernel/Config/Files/FAQ.xml [hunk 3]
```diff
@@ -29,7 +29,7 @@
         <SubGroup>Interface</SubGroup>
         <Setting>
             <Array>
-                <Item>public</Item>
+                <Item Translatable="1">public</Item>
             </Array>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: Strings here should not be translatable as they are the DB names of the state types.

File: Kernel/Config/Files/FAQ.xml [hunk 4]
```diff
@@ -38,7 +38,7 @@
         <Group>FAQ</Group>
         <SubGroup>Core</SubGroup>
         <Setting>
-            <String>FAQ</String>
+            <String Translatable="1">FAQ</String>
         </Setting>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: String should not be translatable as it is the DB name if the root category

File: Kernel/Config/Files/FAQ.xml [hunk 5]
```diff
@@ -221,7 +221,7 @@
         <Group>FAQ</Group>
         <SubGroup>Core</SubGroup>
         <Setting>
-            <String>FAQ#</String>
+            <String Translatable="1">FAQ#</String>
         </Setting>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This string is also not normally translated (compare with Ticket::Hook)

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs,

As you could see in the commit I add some notes on strings that should not be translatable I review until line ~ 200, Please check the remaining strings if its values are used, in such case I think they should not be translated.

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
I removed some translatable flags. If you find more that it should not translated, I will remove them.

File: Kernel/Config/Files/FAQ.xml [hunk 6]
```diff
@@ -99,9 +99,9 @@
         <SubGroup>Core::Explorer</SubGroup>
         <Setting>
             <Hash>
-                <Item Key="30">red</Item>
-                <Item Key="70">orange</Item>
-                <Item Key="100">green</Item>
+                <Item Translatable="1" Key="30">red</Item>
+                <Item Translatable="1" Key="70">orange</Item>
+                <Item Translatable="1" Key="100">green</Item>
             </Hash>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This strings should not be translatable

File: Kernel/Config/Files/FAQ.xml [hunk 7]
```diff
@@ -328,7 +328,7 @@ Your OTRS Notification Master
         <SubGroup>Core::Item</SubGroup>
         <Setting>
             <Hash>
-                <Item Key="Show">public</Item>
+                <Item Translatable="1" Key="Show">public</Item>
                 <Item Translatable="1" Key="Caption">Symptom</Item>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This string should not be translatable

File: Kernel/Config/Files/FAQ.xml [hunk 8]
```diff
@@ -400,9 +400,9 @@ Your OTRS Notification Master
         <SubGroup>Core::LinkObject</SubGroup>
         <Setting>
             <Hash>
-                <Item Key="Object1">FAQ</Item>
-                <Item Key="Object2">FAQ</Item>
-                <Item Key="Type">Normal</Item>
+                <Item Translatable="1" Key="Object1">FAQ</Item>
+                <Item Translatable="1" Key="Object2">FAQ</Item>
+                <Item Translatable="1" Key="Type">Normal</Item>
             </Hash>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This strings should not be translatable

File: Kernel/Config/Files/FAQ.xml [hunk 9]
```diff
@@ -412,9 +412,9 @@ Your OTRS Notification Master
         <SubGroup>Core::LinkObject</SubGroup>
         <Setting>
             <Hash>
-                <Item Key="Object1">FAQ</Item>
-                <Item Key="Object2">FAQ</Item>
-                <Item Key="Type">ParentChild</Item>
+                <Item Translatable="1" Key="Object1">FAQ</Item>
+                <Item Translatable="1" Key="Object2">FAQ</Item>
+                <Item Translatable="1" Key="Type">ParentChild</Item>
             </Hash>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This strings should not be translatable

File: Kernel/Config/Files/FAQ.xml [hunk 10]
```diff
@@ -424,9 +424,9 @@ Your OTRS Notification Master
         <SubGroup>Core::LinkObject</SubGroup>
         <Setting>
             <Hash>
-                <Item Key="Object1">FAQ</Item>
-                <Item Key="Object2">Ticket</Item>
-                <Item Key="Type">Normal</Item>
+                <Item Translatable="1" Key="Object1">FAQ</Item>
+                <Item Translatable="1" Key="Object2">Ticket</Item>
+                <Item Translatable="1" Key="Type">Normal</Item>
             </Hash>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This strings should not be translatable

File: Kernel/Config/Files/FAQ.xml [hunk 11]
```diff
@@ -436,9 +436,9 @@ Your OTRS Notification Master
         <SubGroup>Core::LinkObject</SubGroup>
         <Setting>
             <Hash>
-                <Item Key="Object1">FAQ</Item>
-                <Item Key="Object2">Ticket</Item>
-                <Item Key="Type">ParentChild</Item>
+                <Item Translatable="1" Key="Object1">FAQ</Item>
+                <Item Translatable="1" Key="Object2">Ticket</Item>
+                <Item Translatable="1" Key="Type">ParentChild</Item>
             </Hash>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/FAQ.xml]: This strings should not be translatable

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs,

I add some more comments on strings that should not be translatable.

((enjoy))
cr

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
I removed the translatable flags from strings, that you mentioned. If this PR is able to merge now, then please, sync strings with Transifex after merging.

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
Can you check my PR? I have some other PRs for ITSM packages, too. All of them enhances the translation, so if they are acceptable, can you sync them with Transifex?

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs,

Thank you for update, I will check this pull request ASAP, Regarding the ITSM modules please contact @UdoBretz.

((enjoy))
cr

**Comment by urbalazs** [comment]
Closed for now. See https://github.com/OTRS/FAQ/pull/60

**Pull request closed without merging**